### PR TITLE
Add ShareLinkPopper feature

### PR DESCRIPTION
**Pull request by okucheruk**
This PR add ShareLinkPopper feature.

Screen Shot 2019-09-20 at 8 45 17 AM

### CODE REVIEW

File: app/page/filebrowser/component/FileBrowserListItem.js [hunk 1]
```diff
@@ -189,14 +191,15 @@ export default class FileBrowserListItem {
...
    * @return {FileBrowserListItem} This instance for method chaining.
    */
-  onShare(callback) {
+  _onShare(fileModel) {
     this._shareButton.subscribeToEvent('click', (event) => {
       event.stopPropagation();
       event.preventDefault();
-      callback(this._fileModel.id);
+
+      new ShareLinkPopper(this._shareButton.getNode(), fileModel.downloadURL);
```
Comment (dkalinin) [inline app/page/filebrowser/component/FileBrowserListItem.js]: Request Shared URL from backend.

File: app/page/filebrowser/component/SharLinkPopper.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  /**
+   *
+   * @param {String} link The shareable link value.
+   * @private
+   */
+  _render(link) {
+    const popperInput =
+        new InputBuilder()
```
Comment (dkalinin) [inline app/page/filebrowser/component/SharLinkPopper.js]: Show the beginning of URL.

Comment (okucheruk) [reply app/page/filebrowser/component/SharLinkPopper.js]: Fixed by changing order of `focus()` and  `select()`.

File: app/page/filebrowser/component/FileBrowserListItem.js (hunk 1 above)
Comment (okucheruk) [reply app/page/filebrowser/component/FileBrowserListItem.js]: Refactored.

**Comment by okucheruk** [comment]
@dkalinin, fixed, PTAL.

File: app/page/filebrowser/component/FileBrowserSideBar.js [hunk 3]
```diff
@@ -171,6 +187,10 @@ export default class FileBrowserSideBar {
         .build();
   }
 
+  /**
+   *
+   * @param {String} resourceId
+   */
   showInfo(resourceId) {
```
Comment (dkalinin) [inline app/page/filebrowser/component/FileBrowserSideBar.js]: Please add doc.

Comment (okucheruk) [reply app/page/filebrowser/component/FileBrowserSideBar.js]: Unused method removed.

**Comment by okucheruk** [comment]
Improve UX.

Screen Shot 2019-10-03 at 4 07 59 PM

**Comment by okucheruk** [comment]
@dkalinin, fixed, PTAL.

**Pull request merged**